turbo/transactions/tracing.go

```diff
@@ -224,8 +227,6 @@ func ExecuteTraceTx(
 		if streaming {
 			stream.WriteArrayEnd()
 			stream.WriteObjectEnd()
-			stream.WriteMore()
```

Comment:
don't think you could remove this

Reply:
@zfy0701 can you provide more context?

Reply:
@zfy0701 I suspect you raised this because you have hit this - https://github.com/ledgerwatch/erigon/issues/10376

a short term fix will be included in v 2.60.1 and future v3 for erigon 3
long term proper fix is https://github.com/ledgerwatch/erigon/issues/10389